data/campaigns/Northern_Rebirth/scenarios/12a_Get_the_Gold.cfg

```diff
@@ -574,13 +582,13 @@
             [then]
                 [message]
                     id=Tallin
-                    message= _ "Thanks to Hidel and the elves, we have recovered our gold. Now let’s run down Rakshas and settle up with him once and for all."
+                    message= _ "Thanks to Hidel and the elves, we have recovered our gold. Now let’s run down Rakshas end this once and for all."
```

Comment:
Should there be an 'and' or a comma before the 'end this'?

Reply:
Yep.